randovania/games/samus_returns/logic_database/Area 4 - East.txt

```diff
@@ -366,15 +370,13 @@ Extra - asset_id: collision_camera_007
   * Extra - actor_name: LE_Item_005
   * Extra - actor_type: item_powerbombtank
   > Tunnel to Main Shaft
-      All of the following:
-          Lay Power Bomb
-          Grapple Beam or Can Spiderspark
+      Grapple Beam and Spiderspark (Intermediate) and Can Spiderspark and Lay Power Bomb
```

Comment:
Grapple should still be it's own group because it is an alternative to leave.

As for the Spider spark itself, I'll feel it could be Beginner to escape given its the typical spark shaft, but maybe since it needs a bit more thought, then Intermediate is fine. I'll let you decide.

Reply:
Seems like I grouped grapple incorrectly. As for the spider spark, I do think it could be set to beginner as well. Will change when I get home.